homepage/tests/add_review_search_tests.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+from homepage.models import Course
+from django.db.models import QuerySet
+from pytest_django.asserts import assertTemplateUsed
+
+
+# --------Backend testing-------- #
+@pytest.mark.django_db
+def test_type_get_courses_ordered_by_name():
+    # because an empty string is a substring of all names, all courses returned
+    courses = Course.get_courses_ordered_by_name("")
+

```

Comment:
please remove the blank line.
I would use a blank line as a "separator" if the method is too long and I have too many cases to separate them but in cases of a small method I think we don't have to add it.